### DISCUSSION TITLE
Would love to see a video on how to properly tighten SV06 belts to use with the Ellis profiles

### DISCUSSION THREAD

**Discussion by ryryonline** [bassamanator/Sovol-SV06-firmware#18] [Ideas]
I've watched a few videos on the topic and it's pretty confusing. I found an app that can help tune the belts by "twanging" them, but not sure if that's the correct approach.

**Comment by bassamanator** [comment]
This is the only video that I've found that takes a systematic approach to this: [YT link](https://youtu.be/gB8m6_eGShs). 

Also has a helper page [here](https://prorifi3d.com/pages/how-to-tune-your-timing-belt). Haven't tried it yet.
